CrossRef-REST.js

```diff
@@ -0,0 +1,245 @@
+{
+	"translatorID": "0a61e167-de9a-4f93-a68a-628b48855909",
+	"translatorType": 8,
+	"label": "CrossRef REST",
+	"creator": "Martynas Bagdonas",
+	"target": "",
+	"minVersion": "5.0.0",
```

Comment:
I guess that all special Zotero functions which you use here are also available in version 3.0. Thus the translator is compatible, but it might be that it will not be possible to call it before.